response_operations_ui/controllers/edit_contact_details_controller.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def edit_contact_details(edit_details_data, respondent_id):
+    logger.debug('Validating contact details')
```

Comment:
Probably want to log out `respondent_id` here

Reply:
Also is `validating` the right word for this message? We are past validating here and we're actually changing them?